device/mellanox/x86_64-mlnx_msn3420-r0/ACS-MSN3420/buffers.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers.json.j2
```

Comment:
this is a Spectrum-2 switch, why buffers are taken as Spectrum switch? it should be more like SN3700

Reply:
fixed